web/profiles/joinup/config/install/og.og_role.rdf_entity-collection-facilitator.yml

```diff
@@ -16,21 +16,19 @@ permissions:
   - 'request collection archival'
   - 'update group'
   - 'view any unpublished content'
-  - 'view any unpublished news content'
   - 'create custom_page content'
   - 'edit any custom_page content'
   - 'delete any custom_page content'
   - 'edit any document content'
   - 'delete any document content'
   - 'edit any event content'
   - 'delete any event content'
-  - 'create rdf entity news'
+  - 'create rdf entity content'
+  - 'view any unpublished news content'
   - 'create news content'
   - 'edit any news content'
   - 'delete any news content'
-  - 'update any navigation ogmenu_instance'
-  - 'add new links to og menu instance entities'
```

Comment:
Why are these permissions taken away from the facilitator? Are they not allowed to add OG menu items any more? This is out of scope for this ticket.